specification/cognitiveservices/data-plane/Language/preview/2021-07-15-preview/analyzeconversations.json

```diff
@@ -443,21 +443,21 @@
         }
       }
     },
-    "DeepStackEntityResolution": {
+    "DeepStackExtraInformationBase": {
```

Comment:
Nit: with Prediction you prefaced "Base" i.e. "BasePrediction". Would be nice to be consistent one way or another.